kuksa-client/kuksa_client/cli_backend/__init__.py

```diff
@@ -32,6 +32,7 @@ def __init__(self, config):
         self.cacertificate = config.get('cacertificate', str(self.default_cert_path / 'CA.pem'))
         self.certificate = config.get('certificate', str(self.default_cert_path / 'Client.pem'))
         self.keyfile = config.get('key', str(self.default_cert_path / 'Client.key'))
+        self.tokenfile = config.get('token', str(self.default_cert_path / 'jwt/all-read-write.json.token'))
```

Comment:
Why this is changing the client.
Also I think it is wrong to inject some "default" tokenfile.

@romainletendart  Any comments?

Reply:
I did this to be able to authorize the client to the server but yeah wasn't a good idea. Maybe there is a way to do this more generic.

Reply:
We already set it for grpc right?, see https://github.com/boschglobal/kuksa.val/blob/feature/stresstest/kuksa-client/kuksa_client/cli_backend/grpc.py#L46 Seems similar to what I did in https://github.com/eclipse/kuksa.val/pull/371/files

Reply:
Like Erik said, this is already done in the grpc backend.
If we need it on ws side as well, I think it makes sense to move grpc's code into the common class like you did Lukas.
Just remember to delete the code from the grpc implementation that does the same thing.
Regarding the righteousness of having a default token value, I think this needs to be addressed outside of this PR as we also provide a default private key which is also not secure.

Reply:
Wanted to write a lot, instead put it in #406 

Wrt to this stresstest, it may be a seperate issue, but I do not see why this test can not just have a "clean" config file and give it to the library like a good citizen.
If the lib does not have authorize calls or other suitable hooks anymore, (@romainletendart  ?) maybe we should readd them, they could just do nothing in new GRPC currently

Reply:
We still need to adapt https://github.com/eclipse/kuksa.val/blob/f1e1ce9f5f219a19048d150246bb6562cd969820/kuksa-client/kuksa_client/cli_backend/grpc.py#L45-L47 otherwise we declare twice `self.tokenfile` in grpc's case.
Please note that the one you introduced is a `str` and that grpc expects a `pathlib.Path`.

Reply:
The way you fixed the grpc module should do. 👍